.github/workflows/black.yml

```
@@ -3,6 +3,10 @@ name: Black
 on:
   workflow_dispatch: # Activate this workflow manually
   pull_request:
+  push:
+    branches:
+      # release branches have the form v1.9.x
+      - 'v[0-9].*[0-9].x'
```

If it runs on branches then let's run it on main too.
```suggestion
  push:
    branches:
      - main
      # release branches have the form v1.9.x
      - 'v[0-9].*[0-9].x'
```